packages/react-core/src/components/ProgressStepper/examples/ProgressStepperBasic.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { ProgressStepper, ProgressStep } from '@patternfly/react-core';
+
+export const ProgressStepperBasic: React.FunctionComponent = () => (
+  <ProgressStepper>
+    <ProgressStep
+      variant="success"
+      id="basic-step1"
+      titleId="basic-step1-title"
+      aria-label="completed step, step with success"
+    >
+      First step
+    </ProgressStep>
+    <ProgressStep variant="info" isCurrent id="basic-step2" titleId="basic-step2-title" aria-label="step with info">
```

Comment:
The ids in this example need to be unique - could they reference alignment rather than basic?